src/styles/card-style.scss

```diff
@@ -0,0 +1,122 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.product {
+  justify-content: center;
+  flex-direction: column;
+  display: flex;
+  width: 198px;
+  height: 406px;
```

Comment:
it's not a good practice to hardcode width for elements with content inside. add `padding: 32px 16px 16px`